### PR TITLE
Fixes the bug found by Tom in mean photon per mode

### PR DESCRIPTION
**Pull request by nquesada**
**Description of the Change:**
Fixes the bug in decompositions/graph_embed and adds a test that catches it (test_embed_graph)
**Benefits:**
One less bug one more tests
**Possible Drawbacks:**
~~The test takes about 6 minutes on my pc, because a lot of samples (O(10000)) are needed to catch it~~
**Related GitHub Issues:**

### CODE REVIEW

**Comment by co9olguy** [comment]
Thanks @nquesada for a quick fix

**Comment by co9olguy** [comment]
@nquesada new test seems to be failing. What is it testing for?

File: tests/integration/test_measurement_integration.py [hunk 1]
```diff
@@ -114,3 +114,25 @@ def test_measure_fock(self, setup_eng, cutoff, batch_size):
...
+    @pytest.mark.fixture('pure')
+    def test_embed_graph(self, setup_eng, cutoff, batch_size):
+        """Test that a graph embeded with the right photon number gives the right statistics. """
+        n_samples = 10000
+        eng, prog = setup_eng(2)
+        A = np.array([[0.0, 1.0], [1.0, 0.0]])
+        n_mean_per_mode = 1
+        with prog.context as q:
+            ops.GraphEmbed(A, n_mean_per_mode) | (q[0], q[1])
+            ops.MeasureFock() | (q[0], q[1])
```
Comment (co9olguy) [inline tests/integration/test_measurement_integration.py]: If the goal here is just to test if `GraphEmbed` has the correct mean photon numbers, can we not look at the mean photon numbers directly? e.g., 
```
state = eng.run(prog).state
cov = state.cov()
mean_photon_numbers = some_function(state)
```

Comment (nquesada) [reply tests/integration/test_measurement_integration.py]: yes. That should work much better since it is not stochastic.

Comment (nquesada) [reply tests/integration/test_measurement_integration.py]: The latest commit should do exactly what you requested.

**Comment by co9olguy** [comment]
:thinking: the tests that are failing on travis seem to be passing fine on my local branch

**Pull request merged**